site/source/api/templates.md

```diff
@@ -344,7 +344,8 @@ forms:
 
 <dl>
 {% dtdd name:"<em>eventtype</em>" %}
-Matches a particular type of event, such as 'click'.
+Matches a particular type of event, such as 'click'. Can contain 
+multiple types separated by a `/` eg. 'touchend/mouseup/keyup'
```

Comment:
Maybe dot at the end? Make a full sentence. So, please improve punctuation here. Commas, dots, etc.